src/controllers/titleController.js

```diff
@@ -0,0 +1,51 @@
+const titlecontroller = function (Title) {
+    const getAllTitles = function (req, res) {
+      Title.find({})
+        .then((results) => res.status(200).send(results))
+        .catch((error) => res.status(404).send(error));
+  };
+
+    const getTitleById = function (req, res) {
+      const { titleId } = req.params;
+
+      Title.findById(titleId)
+        .then((results) => res.send(results))
+        .catch((error) => res.send(error));
+    };
+
+    const postTitle = async function (req, res) {
+      const title = new Title();
+
+      title.titleName = req.body.titleName;
+      title.teamCode = req.body.teamCode;
+      title.projectAssigned = req.body.projectAssigned;
+      title.mediaFolder = req.body.mediaFolder;
+      title.teamAssiged = req.body.teamAssiged;
+      // get the shortname
+      const shortnames = title.titleName.split(' ');
+      const shortname = (shortnames[0][0] + shortnames[1][0]).toUpperCase();
```

Comment:
Hi @YHX6, Just want to bring to your attention to line of code. This will not throw an error if the title is only one word.
![Screenshot 2024-03-30 153715](https://github.com/OneCommunityGlobal/HGNRest/assets/49083865/4ddb4e4b-0371-4dd0-868d-d6f9c28d3053)

Reply:
Thanks! I'll fix that